sdk/cosmos/azure-cosmos-spark_3_2-12/src/main/scala/com/azure/cosmos/implementation/SparkBridgeImplementationInternal.scala

```diff
@@ -269,7 +274,7 @@ private[cosmos] object SparkBridgeImplementationInternal extends BasicLoggingTra
     val continuations = tokens
       .map(token => {
         val pkRangeId = token._1
-        val lsn: Long = token._2
+        val lsn: Long = token._2 + 1
```

Comment:
why increase the lsn by 1 here?

Reply:
Added comments - but yes Spark 2 tracks next-to-be-sent continuation - Spark 3 the LSN of the last processed docs - so offset by 1